### PR TITLE
Update redirects in Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -659,7 +659,7 @@ History
 
 ObjectiveFlickr was first released in late 2006. The previous version, 0.9.x,
 has undergone one rewrite and is hosted on [Google 
-Code](http://code.google.com/p/objectiveflickr). It also has a Ruby version
+Code](http://code.google.com/p/objectiveflickr/). It also has a Ruby version
```
Comment (lukhnos) [inline README.md]: I think this should be `https` too.

File: README.md [hunk 2]
```diff
@@ -685,11 +685,11 @@ LFWebAPIKit Copyright (c) 2007-2009 Lukhnos D. Liu and Lithoglyph Inc.
 
 One test in LFWebAPIKit (`Tests/StreamedBodySendingTest`) makes
 use of [Google Toolbox for Mac](
-http://code.google.com/p/google-toolbox-for-mac/), Copyright (c) 2008 Google Inc. Refer to `COPYING.txt` in the directory for the full text of the Apache License, Version 2.0, under which the said software is licensed.
+https://github.com/google/google-toolbox-for-mac), Copyright (c) 2008 Google Inc. Refer to `COPYING.txt` in the directory for the full text of the Apache License, Version 2.0, under which the said software is licensed.
 
 Both ObjectiveFlickr and LFWebAPIKit are released under the MIT license,
 the full text of which is printed here as follows. You can also 
-find the text at: <http://www.opensource.org/licenses/mit-license.php>
+find the text at: <http://opensource.org/licenses/mit-license.php>
```
Comment (lukhnos) [inline README.md]: This should also be `https`.

**Comment by lukhnos** [comment]
@ReadmeCritic Thanks for keeping open source document URLs current and secure. Not sure if yours is a bot account, but I left two comments.

**Comment by ReadmeCritic** [comment]
Will update

**Comment by ReadmeCritic** [comment]
Updated

**Pull request merged**